aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.move

```diff
@@ -199,6 +210,15 @@ module aptos_std::bls12381 {
         verify_aggregate_signature_internal(aggr_sig.bytes, public_keys, messages)
     }
 
+    public fun verify_aggregate_signature_t<T: copy+drop>(
+        aggr_sig: &AggrOrMultiSignature,
+        public_keys: &vector<PublicKeyWithPoP>,
```

Comment:
Note this type is different from what's correponding in `verify_aggregate_signature`.
Is `&vector<PublicKeyWithPoP>` more developer friendly than `vector<PublicKeyWithPoP>`?

Reply:
same question for `items` and across the PR.

Reply:
Let's not add this at all, since it uses the old domain separation.